### PR TITLE
Rename rootModules command to module.callers

### LINKED ISSUE #632: Rename rootModules command interface
Because the internal representation of modules in terraform-ls has changed, the [`rootModules` language server command is going to be updated](https://github.com/hashicorp/terraform-ls/pull/508) to `modules.callers`. In order for the extension to continue to access a list of modules that can be used to call `terraform init`, the command interface will need to be updated to match it.

### PR DESCRIPTION
**Pull request by aeschright**
Pending https://github.com/hashicorp/terraform-ls/pull/508
Closes #632

### CODE REVIEW

**Comment by radeksimko** [comment]
I pushed some more changes, all of which should be explained in the commit message.

One pending question from me is that `relPath` (previously "friendly name") is no longer used anywhere, so I'm inclined to just drop it from the API entirely.

File: src/extension.ts [hunk 1]
```diff
@@ -320,50 +324,45 @@ function execWorkspaceCommand(client: LanguageClient, params: ExecuteCommandPara
...
+interface moduleCaller {
 	uri: string,
-	name: string
+	rel_path: string
 }
 
 interface moduleCallersResponse {
-	moduleCallers: module[],
-	needsInit: boolean
+	v: number,
```
Comment (aeschright) [inline src/extension.ts:333]: Is this the API version? Even though it's not being referenced, let's give it a more descriptive name.

**Comment by aeschright** [comment]
> One pending question from me is that `relPath` (previously "friendly name") is no longer used anywhere, so I'm inclined to just drop it from the API entirely.

Makes sense to me, if the language server is no longer displaying it to the user, there's no reason to share that.

File: src/extension.ts (hunk 1 above)
Comment (radeksimko) [reply src/extension.ts]: yep, updated.

**Comment by radeksimko** [comment]
> Makes sense to me, if the language server is no longer displaying it to the user, there's no reason to share that.

See https://github.com/hashicorp/terraform-ls/pull/525

**Review by radeksimko — DISMISSED** [review]

**Review by radeksimko — APPROVED** [review]

**Pull request merged**